feature-configs/base/ipv6/02_network_attachment_defintion.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: k8s.cni.cncf.io/v1
```

Comment:
I would put these in the `sriov` feature, under [base](https://github.com/openshift-kni/cnf-features-deploy/tree/master/feature-configs/base/sriov) or even under [demo](https://github.com/openshift-kni/cnf-features-deploy/tree/master/feature-configs/demo/sriov)

Reply:
done